### DISCUSSION TITLE
functions to edit repo description and manage tags on mobile site

### DISCUSSION THREAD

**Discussion by NitinIsFit** [community/community#28097] [Other Feature Feedback, Questions, & Ideas] [answered]
GitHub.com mobile site doesn't allow a user to edit repository description and tags.

![Screenshot_20220802-230628.png](https://user-images.githubusercontent.com/94877880/182440432-e099bcfd-d1a3-448a-a5b5-35d844fb187a.png)

If you have to edit repository description and tags, you have to switch to desktop site, which is inconvenient.
Please add these functions on mobile site.

![Screenshot_20220802-231815.png](https://user-images.githubusercontent.com/94877880/182440849-ce501110-67bd-4c78-a748-fd6720df06ef.png)

**Comment by stevepopovich** [comment]
Changing this to general as this is a bug with the website and not the Native mobile app. Thanks for filing!

**Reply by NitinIsFit** [reply]
You gonna fix that? 😐

**Comment by NitinIsFit** [comment]
![Screenshot_20220803-041005.png](https://user-images.githubusercontent.com/94877880/182485838-9c5248cc-b94e-4895-8257-87a13164375a.png)

We cannot rename branches on mobile site.💀

![Screenshot_20220803-040946.png](https://user-images.githubusercontent.com/94877880/182485910-b516b433-a6ad-434d-9f10-b499f6e0ba06.png)

**Comment by NitinIsFit** [comment]
![Screenshot_20220803-042109.png](https://user-images.githubusercontent.com/94877880/182486942-181a65bd-b644-47fa-9240-fb629145e6b7.png)

Cannot delete an issue on mobile too.

**Comment by jagottsicher** [comment]
> Changing this to general as this is a bug with the website and not the Native mobile app. Thanks for filing!

Obviously. But will that be fixed? It's a simple thing to allow to access the desktop version of a page instead of forcing the user to move to another computer.

**Reply by Ambient-Impact** [reply]
While I agree that this should be available in the mobile view, I can access the desktop version in Firefox on Android by toggling on "Desktop Site" in the browser menu. I imagine it will work in any browser that provides that option.

**Reply by tomkimberlin** [reply]
Requesting desktop site does not work for iOS Safari.

**Comment by NitinIsFit** [comment, marked as answer]
Well, I guess after 10 months they have listened to me and made changes into the app to edit repo description, but this is still not available on GitHub.com web for mobile, to be honest GitHub application is making some progress now

**Reply by SottNick** [reply]
i found... or they added it...
![IMG_20241025_224723](https://github.com/user-attachments/assets/4d275937-ff9d-4028-a83d-c9e8b305d7b6)

**Reply by adelobosko** [reply]
2025... Where?
<img width="1834" height="873" alt="image" src="https://github.com/user-attachments/assets/22ad2548-7354-4458-b439-6e7f85a961da" />

**Reply by adelobosko** [reply]
found an answer (for an empty repo need to create first commit or use mobile app) 
https://github.com/orgs/community/discussions/60507#discussioncomment-12484399

**Reply by trusktr** [reply]
Haha, wow, who is ever gonna find that little gear cog on mobile. At least add description to repo General settings under the repo name!
